wsl.go

```diff
@@ -257,9 +271,17 @@ func (p *Processor) parseBlockStatements(statements []ast.Stmt) {
 			leftHandSide                = p.findLHS(stmt)
 			rightHandSide               = p.findRHS(stmt)
 			rightAndLeftHandSide        = append(leftHandSide, rightHandSide...)
+			checkingNilErr              = p.isCheckingErrAgainstNil(stmt, leftHandSide, rightHandSide)
```

Comment:
Hmm, at some point I tried not doing this here and I remember it breaking a lot of other tests.. I thought I needed this to prevent that, but after all my refactoring it seems that's no longer a problem :D